R/calc_views.R

```diff
@@ -88,7 +91,7 @@ which_reals <- function(values, na.value){
   
 }
 set_view_window <- function(views, param, na.value=NA, remove=TRUE, ignore=NULL){
-  view_i <- which(names(views) %in% "view")
+  view_i <- grep('view.', names(views))
```

Comment:
'view.' seems like a magic number that is used in a lot of places, is it possible to make this a constant in some way (not sure if the idea of constants works the same in R packages as in other languages).